src/styles/main.scss

```
@@ -0,0 +1,4 @@
+@import "./untils/vars";
+
+@import "./untils/blocks/Keyframes.scss";

```

```suggestion
@import "./untils/blocks/keyframes.scss";
```
Name files with lowerCase letters